operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/NamespaceChangeable.java

```diff
@@ -0,0 +1,26 @@
+package io.javaoperatorsdk.operator.api.config;
+
+import java.util.Set;
+
+public interface NamespaceChangeable {
```

Comment:
I was actually thinking to do such abstraction, but first I thought that anyways everbody will use `InformerEventSource`. So will add if it is asked for. But actually good that you added, have a feeling of completeness now :)